sdk/AWSSDK.NetFramework.sln

```diff
@@ -0,0 +1,3970 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Since `AWSSDK.NetFramework.sln` was added, shouldn't we also remove [AWSSDK.Net45.sln](https://github.com/aws/aws-sdk-net/blob/dspin/generate-net45-to-net462/sdk/AWSSDK.Net45.sln) and [AWSSDK.Net35.sln](https://github.com/aws/aws-sdk-net/blob/dspin/generate-net45-to-net462/sdk/AWSSDK.Net35.sln) as part of the same PR?

Reply:
We should, but I'm planning to add a step in the generator to delete all "legacy" artifacts (e.g. `bcl35` folders, `Net35` and `Net45` projects, etc...). I didn't want to include more stuff in this PR since it was already too large.